### PR TITLE
Deletion unit + e2e tests

### PR DESCRIPTION
**Pull request by Anishkulkarni9001**
Additional tests to add to e2e test file

### CODE REVIEW

**Review by karthikv6 — DISMISSED** [review]

File: src/scripts/home.js [hunk 1]
```diff
@@ -226,9 +226,11 @@ function delete_profile(profile) {
   req_card.remove();
 
   // Remove profile from list and save list
-  PROFILE_LIST = PROFILE_LIST.filter(
-    (curr_prof) => curr_prof.id !== profile.id
-  );
+  const rmv_idx = PROFILE_LIST.indexOf(profile);
+  console.log("rmv idx: " + rmv_idx);
```
Comment (Sirajpal) [inline src/scripts/home.js:230]: Can't this be deleted

**Review by Sirajpal — COMMENTED** [review]
looks good, but there is a console.log statement left over. Otherwise, looks compatible with the e2e branch.

File: src/scripts/home.js (hunk 1 above)
Comment (benjxia) [reply src/scripts/home.js:230]: i removed it while resolving merge conflicts, it hasnt been pushed yet

**Review by Sirajpal — APPROVED** [review]

**Comment by benjxia** [comment]
ight stop approving stuff without reviewing

**Comment by benjxia** [comment]
this branch fails every test

**Review by benjxia — APPROVED** [review]
looks good now

**Review by chrisliang12 — APPROVED** [review]

**Pull request merged**